portal/settings.py

```diff
@@ -242,3 +245,5 @@
 else:
     # Otherwise, send emails to the console
     EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+DEBUG = True 
```

Comment:
```suggestion

```

No need to explicitly define DEBUG as it is already defined above in the file.


you just need to export it as an environment variable to use it.